policy/openbot/train.py

```diff
@@ -6,6 +6,8 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import tensorflow as tf
+import wandb
+from wandb.keras import WandbCallback
```

Comment:
Can we make using weights&biases optional by passing a flag?

Reply:
this only shows a warning if not configured - as I remember
but I can refactor it if needed

Reply:
The main issue is that if wand is installed it requires user input from the terminal. Most users that use the web interface would not pay attention to the terminal. So I think we could add an arg to the train script to disable. Also we could add the setting to web interface to configure wand (or is this too much work)?

Reply:
@sanyatuning would be great if you can refactor this. 
- add flag train script so it can be disabled
- option in web interface to setup wand or disable it

Reply:
Flag added. No option in web interface.